src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+   
+    <div class="card" data-qa="card">
+      <img 
+        class="card__rectangle" 
+        src="./images/imac.jpeg" 
+        alt="APPLE"
+      >
+      <div class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+      <div class="review card__review">
+        <div class="review__stars-container">
+          <div class="stars review__stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <div class="review__text">Reviews: 5</div>
+      </div>  
+      <div class="price card__price">
+        <div class="price__text">Price:</div>
+        <div class="price__value">$2,199</div>
+      </div>
+      <div class="card__button">
```

Comment:
It kind of looks like we have multiple BEM blocks right now. We should have only two - card and stars (no `review` and `price` block).